public/views/user/user.html

```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="col-sm-3">
-    <img src="/images/rick.jpg" class="user-profile-image" />
+    <img ng-src="{{ UserCtrl.user.avatar150 }}" class="user-profile-image" />
```

Comment:
We pass the user DB model to the user controller, so I think this should be `user.image`